samples/Calculator/csharp/windows/calculator/Package.appxmanifest

```diff
@@ -7,15 +7,15 @@
   IgnorableNamespaces="uap mp">
 
   <Identity
-    Name="b3e4af42-aec3-46e8-a138-cb2e481514b2"
+    Name="7657a35a-6073-4280-b14b-409220cbd347"
     Publisher="CN=React Native Windows Sample"
     Version="1.0.0.0" />
 
-  <mp:PhoneIdentity PhoneProductId="b3e4af42-aec3-46e8-a138-cb2e481514b2" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+  <mp:PhoneIdentity PhoneProductId="7657a35a-6073-4280-b14b-409220cbd347" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
```

Comment:
Ideally the package identity is stable across updates. If you create from a fresh app template you'll need to copy the project and package ids back over.

Reply:
Is this for both `Name` and `PhoneIdentity`? Or just `PhoneIdentity`?